src/main/java/donnafin/ui/UiManager.java

```diff
@@ -183,10 +178,8 @@ public ViewFinderState getUiState() {
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         CommandResult commandResult = logic.execute(commandText);
         Consumer<Ui> uiAction = commandResult.getUiAction();
-        assert !uiAction.equals(null) : "commandResult.uiAction was set as null";
-        if (!uiAction.equals(null)) {
-            uiAction.accept(this);
-        }
+        assert uiAction != null : "commandResult.uiAction was set as null";
```

Comment:
Theres 1 more in logic manager. I refactor that later?

Reply:
sure